### ISSUE TITLE
ignore helm buffers in winner mode

### ISSUE THREAD

**Issue by danielwuz** [syl20bnr/spacemacs#417]
Does it make sense to ignore `helm` buffers in winner mode? Undo a frame of which helm buffer in is boring because that buffer is not interactive.

I had something like below in my dotspacemacs.
```
  (setq winner-boring-buffers
        (append winner-boring-buffers '("*helm mini*" "*helm projectile*" "*helm M-x*" "*helm resume*")))
```
but I think it's really for everyone.

**Comment by tuhdo** [comment]
Definitely. Helm buffers are used for resuming later. `helm-mini` ignores them by default. You may want to look at [this winner-mode configuration](https://github.com/thierryvolpiatto/emacs-tv-config/blob/master/.emacs.el#L1706) of Thierry, current Helm maintainer. At the minimum, this is his regexp to exclude all Helm buffers:

```elisp
(defvar winner-boring-buffers-regexp "\\*[hH]elm.*")
```

**Comment by syl20bnr** [comment]
Very nice ! Thank you both. We can go with the regexp.

**Comment by syl20bnr** [comment]
Anybody for submitting a PR for this handy tweak ?

**Comment by syl20bnr** [comment]
Fixed by #582
